### DISCUSSION TITLE
v0.0.36

### DISCUSSION THREAD

**Discussion by thliebig** [thliebig/openEMS-Project#132] [Announcements]
This is the release of v0.0.36

* Windows build using the MSVCv14
* Allows build of python modules, builds for python 3.10 and 3.11 included

[See Windows Python Install Instructions](https://docs.openems.de/python/install.html#windows)

<hr /><em>This discussion was created from the release <a href='https://github.com/thliebig/openEMS-Project/releases/tag/'>v0.0.36</a>.</em>

**Comment by Brody-stack** [comment]
Could not find "InitOpenEMS.m" file
